drivers/storage/efs/tests/efs_test.go

```diff
@@ -22,56 +22,54 @@ import (
 
 	// load the driver
 	"github.com/codedellemc/libstorage/drivers/storage/efs"
-	efsx "github.com/codedellemc/libstorage/drivers/storage/efs/executor"
+	efsUtils "github.com/codedellemc/libstorage/drivers/storage/efs/utils"
 )
 
 var (
 	configYAML = []byte(`
+libstorage:
+  service: efs
+  integration:
+    volume:
+      operations:
+        mount:
+          preempt: true
 efs:
-  tag: %s
-  region: %s
-  securityGroups: %s
+  region: us-west-2
+  endpoint: ec2.us-west-2.amazonaws.com
+  accessKey: %s
+  secretKey: %s
```

Comment:
Just as a general question across all drivers, and something I've been thinking about anyways, is how much of the config do we really want codified in these test files? It seems to be as little as possible, and leaving it to environment variables is best. I would put in reasonable defaults here where it is helpful, but anything where someone *has* to make a change, I'd rather not see it here at all.  So rather than doing `os.getenv(AWS_ACCESSKEY)` down below, I'd rather see the tests run something like

`EFS_ACCESSKEY=$AWS_ACCESSKEY ./efs.test`

Thoughts?